environments/icds-cas/public.yml

```diff
@@ -124,7 +124,7 @@ localsettings:
     - 127.0.0.1
     - '{{ SITE_HOST }}'
     - '{{ CAS_SITE_HOST }}'
-    - '{{ hostvars[groups.proxy.0].public_ip }}'
+      # - '{{ hostvars[groups.proxy.0].public_ip }}' TODO: hostvars seems to not work?
```

Comment:
I don't think any hosts have this var set

Reply:
erg. I realize what I did now after seeing https://github.com/dimagi/commcare-cloud/pull/2050/commits/6da8fe85ba921e3b136569490f6da1ed2bd6bd35

I didn't fully add the comma to all lines... totally missed htat yesterday. thanks for fixing